ccd-definition/CaseEventToFields/DefendantResponse-prod.json

```diff
@@ -16,7 +16,7 @@
     "CaseFieldID": "respondent1DQRequestedCourt",
     "PageDisplayOrder": 15,
     "PageFieldDisplayOrder": 2,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "COMPLEX",
     "PageID": "RequestedCourt",
     "PageShowCondition": "multiPartyResponseTypeFlags=\"FULL_DEFENCE\"",
     "FieldShowCondition": "respondent1ClaimResponseType=\"FULL_DEFENCE\" OR respondent1ClaimResponseTypeToApplicant2=\"FULL_DEFENCE\"",
```

Comment:
1. Why "DisplayContext" is changed to "COMPLEX"?
2. Can both the fields be moved to ccd-definition/CaseEventToFields/DefendantResponse.json and this file be deleted please.